lib/renderer/renderer-webpack.js

```diff
@@ -153,58 +174,75 @@ class Renderer {
         }
     }
     /**
-     * @param {object[]} files
+     * @param {object[]} filepaths
      */
-    async _loadBundleFilesToMemory(files) {
+    async _loadBundleFilesToMemory(filepaths) {
         const vm = this;
-        vm.mfs.mkdirpSync("/expressvue/bundles");
-        files.forEach(file => {
-            vm.mfs.writeFileSync(`/expressvue/bundles/${path.parse(file).base}`, fs.readFileSync(file));
+        filepaths.forEach(filepath => {
+            const bundlePath = this.getBundleFilePath(filepath);
+            vm.mfs.mkdirpSync(bundlePath.memory.base);
+            vm.mfs.writeFileSync(`${bundlePath.memory.path}.js`, fs.readFileSync(filepath));
```

Comment:
Since the function is async, I think you should use the async versions of `vm.mfs.mkdirp` and `vm.mfs.writeFile` if those exist.

Reply:
mfs doesn't have promises, its either callbacks or the sync versions